gobblin-modules/gobblin-sql/src/main/java/org/apache/gobblin/writer/commands/BaseJdbcBufferedInserter.java

```diff
@@ -51,6 +51,7 @@
   private static final Logger LOG = LoggerFactory.getLogger(BaseJdbcBufferedInserter.class);
 
   protected static final String INSERT_STATEMENT_PREFIX_FORMAT = "INSERT INTO %s.%s (%s) VALUES ";
+  protected static final String REPLACE_STATEMENT_PREFIX_FORMAT = "REPLACE INTO %s.%s (%s) VALUES ";
```

Comment:
`REPLACE` is non-standard-SQL, specific to Mysql, so it shouldn't live in the base.  other DBs may support a `MERGE` statement... or have their own extensions.